pkg/export/alloy/traces.go

```diff
@@ -31,6 +32,9 @@ func TracesReceiver(
 		tr := &tracesReceiver{
 			cfg: cfg, hostID: ctxInfo.HostID, spanMetricsEnabled: spanMetricsEnabled,
 			input: input.Subscribe(),
+			is: instrumentations.NewInstrumentationSelection([]string{
```

Comment:
We need to fix this in the future to make it compatible with Alloy.

Reply:
As discussed, we'll follow up with a fix here when we add configuration for Alloy.